test/include/userobjects/GetMaterialPropertyBoundaryBlockNamesTest.h

```diff
@@ -42,6 +42,15 @@ GetMaterialPropertyBoundaryBlockNamesTest::performTest(const std::vector<T> & re
   // Extract the expected names
   std::vector<std::string> expected_names = getParam<std::vector<std::string>>("expected_names");
 
+  _console << "Expected:";
+  for (const auto & v : expected_names)
+    _console << " " << v;
+  _console << std::endl;
+  _console << "Retrieved:";
+  for (const auto & v : retrieved_names)
+    _console << " " << v;
+  _console << std::endl;
+
```

Comment:
It does not hurt in a test object. This is how I know how to update that test. Do you still want this to be removed?

Reply:
Yes. It obfuscates what is being tested. Console output is not being tested but adding this console output suggests that it might be, which is misleading

Reply:
removed.